packages/prosess/prosess-vedtak/src/VedtakProsessIndex.spec.tsx

```diff
@@ -15,9 +15,33 @@ const {
   InnvilgetEngangsstønadTilGodkjenningForSaksbehandlerUtenOverstyring,
   InnvilgetRevurderingForeldrepengerTilGodkjenningForSaksbehandlerUtenOverstyring,
   GodkjentForeldrepengerMedManueltBrevForSaksbehandlerMedOverstyring,
+  LegacyOverstyring,
+  LegacyOverstyringHarSendtTilbakeFraBeslutter,
 } = composeStories(stories);
 
 describe('<VedtakProsessIndex>', () => {
+  const oldWindowMatchMedia = window.matchMedia;
+
+  beforeAll(() => {
+    Object.defineProperty(window, 'matchMedia', {
+      writable: true,
+      value: vi.fn().mockImplementation(query => ({
+        matches: false,
+        media: query,
+        onchange: null,
+        addListener: vi.fn(), // Deprecated
+        removeListener: vi.fn(), // Deprecated
+        addEventListener: vi.fn(),
+        removeEventListener: vi.fn(),
+        dispatchEvent: vi.fn(),
+      })),
+    });
+  });
+
+  afterAll(() => {
+    window.matchMedia = oldWindowMatchMedia;
+  });
```

Comment:
kan vurdere å bytte denne med `vi.hoisted`i denne filen og i `FritekstRedigeringModal.spec.tsc`
```tsx
vi.hoisted(() => {
  Object.defineProperty(window, 'matchMedia', {
    writable: true,
    value: vi.fn().mockImplementation(query => ({
      matches: false,
      media: query,
      onchange: null,
      addListener: vi.fn(), // Deprecated
      removeListener: vi.fn(), // Deprecated
      addEventListener: vi.fn(),
      removeEventListener: vi.fn(),
      dispatchEvent: vi.fn(),
    })),
  });
});
```
Effekten er forøvrig den samme, men er kanskje hakket ryddigere: 
https://rebeccamdeprey.com/blog/mock-windowmatchmedia-in-vitest

Reply:
Har endra til dette. Var ikkje klar over hoisted